docs/references/content-as-code.mdx

```diff
@@ -0,0 +1,47 @@
+# Content as code
+
+You can download your charts and dashboards as code, make changes, then upload your changes back to your Lightdash project using the CLI. Content as code is also useful for creating new Lightdash projects where you want the content to be copied from an existing project and added to your new project.
+
+## `lightdash download` downloads content as code
+
+From the [Lightdash CLI](/references/lightdash-cli), you can use the command `lightdash download` to download all of the charts and dashboards from your Lightdash project as code. All of the charts and dashboards will be written as .yml files to a `lightdash` directory inside of your dbt project. 
+
+#### Running `lightdash download` will overwrite any changes you have locally
+
+For example:
+- I run `lightdash download` and one of the charts that is downloaded is `emea-revenue-per-month.yml`
+- I make some changes to the `emea-revenue-per-month.yml` file and save them
+- I **do not** upload my changes, they are just saved locally
+- I run `lightdash download` again 
+- The changes I made to `emea-revenue-per-month.yml` will be overwritten by the latest chart version downloaded from the Lightdash application
+
+#### Content that's been downloaded as code can still be updated in the Lightdash application
+
+For example:
+- There is a Lightdash project called `Stellar Marketing`
+- Priyanka runs `lightdash download` and downloads all of the project's content as code, including a chart called `Total new clients`
+- Jake opens the `Total new clients` chart in the Lightdash application and makes some changes
+- Priyanka doesn't run `lightdash download`, so the `total-new-clients.yml` chart that Priyanka has as code is the old version of the chart, before Jake updated it. 
+- Priyanka makes changes to `total-new-clients.yml` then uploads her changes and overwrites the changes that Jake made in the Lightdash application. 
+- Both Jake and Priyanka can update the same chart as code, or in the Lightdash application. 
+
+## `lightdash upload` updates or creates any content as code to your project
+
+From the [Lightdash CLI](/references/lightdash-cli), you can use the command `lightdash upload` to upload all of the charts and dashboards saved as .yml in your `lightdash/` directory in your dbt project to the Lightdash application. 
```

Comment:
> upload all

Only updated content, I know you explain it below, but it is not clear until you real the use case